drivers/modem/gsm_ppp.c

```diff
@@ -1,5 +1,7 @@
 /*
  * Copyright (c) 2020 Intel Corporation
+ * Copyright (c) 2020 Laird Connectivity
```

Comment:
Why are you adding this Laird Connectivity copyright from 2020 here?

Reply:
@ycsin Basically the timestring conversion is copied from the hl7800 driver, with some modifications